### ISSUE TITLE
[deploy] Polish logging for deployment to make what's happening towards the end a bit clearer

### ISSUE THREAD

**Issue by joshlong** [jreleaser/jreleaser#1747]
Hi, 

the logs are a little confusing at the very end of a deployment. not sure what the `attempt 1`, `attempt 2`, then `attempt 1`, and then `attempt 2` are in reference to. 

```
[INFO]     [sign] target/staging-deploy/com/joshlong/lucene-spring-boot-starter/0.0.14/lucene-spring-boot-starter-0.0.14-sources.jar
[INFO]     [sign] target/staging-deploy/com/joshlong/lucene-spring-boot-starter/0.0.14/lucene-spring-boot-starter-0.0.14.jar
[INFO]     [mavenCentral]  - com.joshlong-lucene-spring-boot-starter-root-0.0.14-bundle.zip
[INFO]     [mavenCentral] Attempt 1 of 61
[INFO]     [mavenCentral] Attempt 2 of 61
[INFO]     [mavenCentral] Attempt 1 of 61
[INFO]     [mavenCentral] Attempt 2 of 61
[INFO]     [mavenCentral] Attempt 3 of 61
[INFO]     [mavenCentral] Attempt 4 of 61
[INFO]     [mavenCentral] Attempt 5 of 61
[INFO]     [mavenCentral] Attempt 6 of 61
```

thanks for this amazing project!

**Comment by Harsh4902** [comment]
@aalmiray I want to work on this issue. Can you please tell how to reproduce it?

**Comment by aalmiray** [comment]
@Harsh4902 you have to follow the instructions found at https://jreleaser.org/guide/latest/examples/maven/maven-central.html to publish to Maven Central using the `mavenCentral` deployer.

What's missing is a log statement with `info` level at https://github.com/jreleaser/jreleaser/blob/d9665640eaab39833aa337820ff59c0d987b102e/sdks/jreleaser-mavencentral-java-sdk/src/main/java/org/jreleaser/sdk/mavencentral/MavenCentral.java#L127-L132 before `api.publish() is invoked

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.15.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.15.0))
